### PR TITLE
add Ostentatious Arrival to spellshape localization 

### PR DESCRIPTION
**Pull request by ammalagonc**
also, brush up Extend Boost feat.

### CODE REVIEW

File: packs/feats/extend-boost.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You can increase the duration of your eidolon's boosts.</p>\n<p>You learn the <em>extend boost</em> link spell. Increase the number of Focus Points in your focus pool by 1.</p>"
+            "value": "<p>You can increase the duration of your eidolon's boosts.</p>\n<p>You learn the <em>@UUID[Compendium.pf2e.spells-srd.Item.Extend Boost]</em> link spell. Increase the number of Focus Points in your focus pool by 1.</p>"
```
Comment (TikaelSol) [inline packs/feats/extend-boost.json:14]:
```suggestion
            "value": "<p>You can increase the duration of your eidolon's boosts.</p>\n<p>You learn the @UUID[Compendium.pf2e.spells-srd.Item.Extend Boost] link spell. Increase the number of Focus Points in your focus pool by 1.</p>"
```

File: packs/feats/ostentatious-arrival.json [hunk 2]
```diff
@@ -24,7 +24,103 @@
...
+                        "label": "{item|name}",
+                        "value": "ostentatious-arrival"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "alwaysActive": true,
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Prompt.DamageType",
```
Comment (TikaelSol) [inline packs/feats/ostentatious-arrival.json:75]: This should probably get its own label to be clear what its for

Comment (ammalagonc) [reply packs/feats/ostentatious-arrival.json]: I added a new prompt type. Please check when you get a moment.

Thanks!

File: static/lang/re-en.json [hunk 3]
```diff
@@ -3661,6 +3661,7 @@
                 "Color": "Select a Color.",
                 "ConsciousMind": "Select a conscious mind.",
                 "CreatureSize": "Select a creature size.",
+                "CreatureTrait": "Select the summoned or manifested creature trait.",
```
Comment (TikaelSol) [inline static/lang/re-en.json:3664]: That's really long for a roll option label, Something like "Ostentatious Arrival Creature Trait" would work instead.

Comment (ammalagonc) [reply static/lang/re-en.json]: Alright, changed - check when you get a moment.

File: static/lang/re-en.json [hunk 4]
```diff
@@ -3661,6 +3661,7 @@
                 "Color": "Select a Color.",
                 "ConsciousMind": "Select a conscious mind.",
                 "CreatureSize": "Select a creature size.",
+                "CreatureTrait": "Ostentatious Arrival Creature Trait.",
```
Comment (TikaelSol) [inline static/lang/re-en.json:3664]: This this out of the generic prompt area

Comment (ammalagonc) [reply static/lang/re-en.json]: Moved

It's now under Summoner > OstentatiousArrival >Prompt

File: packs/feats/ostentatious-arrival.json [hunk 5]
```diff
@@ -24,7 +24,103 @@
...
+                        "label": "{item|name}",
+                        "value": "ostentatious-arrival"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "alwaysActive": true,
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Summoner.OstentatiousArrival.Prompt",
```
Comment (TikaelSol) [inline packs/feats/ostentatious-arrival.json:75]:
```suggestion
                "label": "PF2E.SpecificRule.Summoner.OstentatiousArrival.Label",
```

File: static/lang/re-en.json [hunk 6]
```diff
@@ -4469,6 +4471,11 @@
                     "Prompt": "Select an element."
                 }
             },
+            "Summoner": {
+                "OstentatiousArrival": {
+                    "Prompt": "Ostentatious Arrival Creature Trait."
```
Comment (TikaelSol) [inline static/lang/re-en.json:4476]:
```suggestion
                    "Label": "Ostentatious Arrival: Creature Trait."
```

**Pull request merged**